### PR TITLE
Cleanup master and move stuff

### PR DESCRIPTION
**Pull request by drewvolz**

### CODE REVIEW

File: app/.gitignore [hunk 1]
```diff
@@ -1,9 +1,7 @@
 # OSX
-#
```
Comment (hawkrives) [inline app/.gitignore:2]: I would probably recommend against editing this file; when you go to upgrade React Native, the `react-native upgrade` command shows you a diff between the local and upgraded copies of the files, so the fewer files you edit the fewer changes you have to look through.

File: app/.gitignore [hunk 2]
```diff
@@ -39,3 +35,7 @@ buck-out/
 \.buckd/
 android/app/libs
 android/keystores/debug.keystore
+
+# KEYS
+keys.js
+.env.js
```
Comment (hawkrives) [inline app/.gitignore:41]: Do you need both of these?

Comment (drewvolz) [reply app/.gitignore:41]: I asked you about that! You said no one looks at the .gitignore so it didn't matter.

...But I mean, no, neither are necessary. Currently not using either of those.

File: app/.gitignore (hunk 1 above)
Comment (drewvolz) [reply app/.gitignore:2]: Good call. I'll take your advice.

File: app/.gitignore (hunk 2 above)
Comment (hawkrives) [reply app/.gitignore:41]: Oh, I thought you were asking about, like, Xcode project files! 

It doesn't _matter_, per se, no, but I like the feeling when you know what each line of your `.gitignore` is ignoring.

**Pull request closed without merging**